terraform/services/config/tofu.tf

```
@@ -0,0 +1,33 @@
+variable "region" {
+  default  = "us-east-1"
+  nullable = false
+  type     = string
+}
+
+variable "secondary_region" {
+  default  = "us-west-2"
+  nullable = false
+  type     = string
+}
+
+provider "aws" {
+  region = var.region
```

Drop variables in this file and set region here


```suggestion
  region = "us-east-1"
```